### DISCUSSION TITLE
[Question]: Link to a specific time in the archive?

### DISCUSSION THREAD

**Discussion by khseal** [blakeblackshear/frigate#12778] [Ask A Question] [answered]
### What is your question:

Is it possible to open the camera archive for a certain time using a link?
This is necessary for the telegram bot to follow the link from it to the frigat to the event.

**Comment by NickM-27** [comment, marked as answer]
You can jump to a specific review item using the ?id query arg with a review item id

**Comment by khseal** [comment]
Thanks It works.
